packages/@aws-cdk/custom-resource-handlers/lib/custom-resources-framework/utils/framework-utils.ts

```diff
@@ -13,6 +13,9 @@ export function buildComponentName(fqn: string, type: ComponentType, entrypoint:
 
 export function toLambdaRuntime(runtime: Runtime) {
   switch (runtime) {
+    case Runtime.NODEJS_20_X: {
```

Comment:
What do you think about adding a `LATEST` in here and pointing that to node20 as well?

Reply:
makes sense after pointing the DEFAULT_RUNTIME  in framework to Runtime.NODEJS_LATEST, next time when we upgrade we'll only need to cover another type in enum and add it here, framework should pick the latest version from this value. 
Updated in latest commit for review.